src/bootstrap.loader.js

```diff
@@ -0,0 +1,98 @@
+/* eslint func-names: 0 */
+
+import loaderUtils from 'loader-utils';
+
+import resolveModule from './utils/resolveModule';
+import checkBootstrapVersion from './utils/checkBootstrapVersion';
+import joinLoaders from './utils/joinLoaders';
+import extractStylesLoader from './utils/extractStylesLoader';
+import createRequire from './utils/createRequire';
+import logger from './utils/logger';
+import { bootstrapVersion, loglevel, createConfig } from './bootstrap.config';
+
+module.exports = function() {};
+
+module.exports.pitch = function(source) {
+  if (this.cacheable) this.cacheable();
+
+  global.__DEBUG__ = loglevel === 'debug' || process.env.DEBUG === '*';
+
+  logger.debug(`Hey, we're in DEBUG mode! Yabba dabba doo!`);
+
+  logger.debug('Usung Bootstrap version:', bootstrapVersion);
```

Comment:
I think we shouldn't spam anything to console in default mode, if something went wrong — we will throw. And the real use case I can make up for it to be enabled is when user has faced some issue and we will ask him to turn debug mode on -> post output to help us find the problem. So debug is apt term here, what do you think?